lib/spree_sam/notifications/providers/parse.rb

```diff
@@ -0,0 +1,28 @@
+require "httparty"
+
+module SpreeSam
+  module Notifications
+    module Providers
+      class Parse
```

Comment:
I really liked this!

quick note:
if you are going to build the possibility of specifying different providers, let's make a parent class with all the shared methods and the ones that have to be implemented, for example: a method build_payload, which 'decorate' the hash payload to customize params required by the strategy